src/test/java/uk/gov/hmcts/reform/sandl/snlapi/controllers/HearingPartControllerTest.java

```diff
@@ -29,6 +29,9 @@
 public class HearingPartControllerTest {
 
     private static String HEARINGS_URL = "/hearing-part";
+    private static String HEARINGS_WITH_IS_LISTED_PARAM_URL = "/hearing-part?isListed=false";
+    private static String HEARINGS_WITH_WRONG_LISTED_PARAM_URL = "/hearing-part?isListed=";
```

Comment:
Do we need to have a test for isListed=true?

Reply:
In this project, we just pass a request to events project. For true/false the way is the same. True/false cases are tested here in events: https://github.com/hmcts/snl-events/pull/96/files#diff-4b4bdc559ea6dc0c2aeb841b41919d56